content/docs/setup/kubernetes/minimal-install/index.md

```diff
@@ -15,21 +15,15 @@ Refer to the [prerequisites](/docs/setup/kubernetes/quick-start/#prerequisites)
 
 ## Installation steps
 
-1. If using a Helm version prior to 2.10.0, install Istio's [Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions)
-via `kubectl apply`, and wait a few seconds for the CRDs to be committed in the kube-apiserver:
-
-    {{< text bash >}}
-    $ kubectl apply -f install/kubernetes/helm/istio/templates/crds.yaml
-    {{< /text >}}
-
-1. Choose one of the following two
-**mutually exclusive** options described below.
+Choose one of the following two **mutually exclusive** options described below.
```

Comment:
How is the reader supposed to choose? On what should they base the decision?

Reply:
@rcaballeromx how about the following with some cleanup:
```
If you prefer to use Kubernetes manifests to deploy Istio, use option #1 below.  If you want Helm's TIller Pod(link to technical description of what TIller is) to manage your Istio release, please use Option 2 below.
```

Reply:
@linsun ya that should probably be a `>` or an info emoji.

Reply:
I liked @sdake version because it clarifies under what circumstances one options should be chosen over the other without forcing the reader to scroll down to make the decision. Ideally, I would introduce the options in a list and link to the appropriate heading in the text. One possibility would be:
```suggestion
You have two mutually exclusive options to install Istio:
- To use Kubernetes manifests to deploy Istio,  follow the instructions for [option #1](#option1).
- To use [Helm's Tiller pod](link to technical description of what TIller is) to manage your Istio release, follow the instructions for [option #2](#option2).
```